webpack.config.js

```diff
@@ -39,8 +39,8 @@ const extractConfig = {
 const entryPointNames = [
 	'element',
 	'i18n',
-	'components',
 	'utils',
+	'components',
```

Comment:
I think the order is defined by the dependencies declaration in the PHP side, isn't it?